pygluu/kubernetes/create.py

```diff
@@ -69,6 +71,11 @@ def main():
         parser.print_help()
         return
     copy_templates()
+
+    # Not sure if we intercept the gui installation from here
+    if args.subparser_name == "gui-install":
+        app.run(host='0.0.0.0', port=5000, debug=True)
```

Comment:
There should be a way to enable/disable `debug`, i.e. `pygluu-kubernetes gui-install --debug`. In production, this `debug` flag must be disabled.

Reply:
@iromli do you prefer using argument or overriding the env vars?

Reply:
I would say option / argument in this case (aligned with others commands).